jenkins.sh

```diff
@@ -19,6 +19,7 @@ export LIBPROCESS_IP=127.0.0.1
 
 # Needed for integrative provisioner tests
 export CGCLOUD_ME=jenkins@jenkins-master
+export TOIL_AWS_KEYNAME=jenkins@jenkins-master
```

Comment:
From the header comment above

> A PR may *temporarily* modify this file but a PR will only be merged if this file is identical
 between the PR branch and the target branch

Do you know if this will be an issue when trying to merge this PR?